MAILSERVER.md

```diff
@@ -1,75 +0,0 @@
-# Description
-
-A poorly named "Mailserver" is essentially a Whisper node that stores message history in either a LevelDB or PostgreSQL database.
-
-A Status app user can run their own Mailserver for faster message retrieval or additional security.
```

Comment:
Why not just move this and `BOOTNODE.md` to `_docs`?

Reply:
Well, `BOOTNODE.md` is about running `cmd/bootnode`, doesn't make sense to keep it if we remove the app.

`MAILSERVER.md` runs `cmd/statusd` (which stays for now), so I suppose we could keep it indeed. Though it basically shouldn't be used at this point. If someone needs to run a Mailserver, it's better to checkout some older version, running anything WakuV1 from current state has no guarantees to be correct.

@jakubgs do we have plans to kick `notify.prod`?

Reply:
As far as I know we can't unless we implement support for Gorush directly in `nim-waku`, and that's not happening anytime soon.

I'm fine with removing `bootnode` stuff.

Reply:
> @jakubgs: Mailservers(waku v1 nodes) are still being used in notify.prod fleet,

Why is notify.prod using wakuv1? Is this fleet used only by those users that have not updated their clients to more recent versions?

Reply:
Yeah I also realised that `notify.prod` is not related to `Mailserver` 🤔 
`Mailserver == Storenode`, I thought it has nothing to do with the push notifications server

Reply:
Sorry @richard-ramos, it's actually using V2:
```yaml
status_go_waku_v1_enabled: false
status_go_waku_v2_enabled: true
```
https://github.com/status-im/infra-push-notify/blob/b9a02e093a9de2e028a3d50f1a61015c02a57390/ansible/group_vars/notify.yml#L26-L27

@igor-sirotin you're right, the `push.prod` fleet has no storing enabled:
```yaml
status_go_history_enabled: false
```
https://github.com/status-im/infra-push-notify/blob/b9a02e093a9de2e028a3d50f1a61015c02a57390/ansible/group_vars/notify.yml#L25

So it should be fine.